main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+## Definition of "specific"
+
+In the context of ERTP and assets, "specific" means "non-fungible".
+
+Assets of the same assay are not fungible if each different item of the assay
+cannot be replaced with another.
+House ownership rights are not fungible, because each ownership right refers to a 
+specific house and one house is not equal to another.
+
+## Creating a fungible asset with ERTP
+
+### Creating and sending the asset
+
+In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+gives you the power to create more digital assets of the same type at
+will.
+
+Let's say we own an Opera and want to sell tickets to seats for ballet shows. Tickets are
+the specific assets we want to represent. Tickets refer to a specific seat for a specific 
+show at a specific time and date.
+
+To do that, you would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
+(`npm install @agoric/ertp`) and then:
+
+```js
+import { makeMint } from '@agoric/ertp/core/mint';
+
+import harden from '@agoric/harden';
+
+import { noCustomization } from '@agoric/ertp/core/config/noCustomization.js';
+import { makeCoreMintKeeper } from '@agoric/ertp/core/config/coreMintKeeper';
+
+import { insist } from '@agoric/ertp/util/insist';
+import { mustBeComparable } from '@agoric/ertp/util/sameStructure';
+
+const insistOptDescription = optDescription => {
+  insist(!!optDescription)`optDescription must be truthy ${optDescription}`;
+  mustBeComparable(optDescription);
+};
+
+function makeInviteConfig() {
+  return harden({
+    ...noCustomization,
+    makeMintKeeper: makeCoreMintKeeper,
+    extentOpsName: 'uniExtentOps',
+    extentOpsArgs: [insistOptDescription],
+  });
+}
+
+const balletTicketMint = makeMint('Agoric Ballet Opera tickets', makeInviteConfig);
+```
+
+At this Opera, there are [1114](https://fr.wikipedia.org/wiki/Grand_Th%C3%A9%C3%A2tre_(Bordeaux)#Salle_de_spectacle) seats numbered `1` to `1114`.
+Objects that represent valid tickets will have the following properties:
+- `seat` with a number
+- `show` with a string describing the show
+- `start` with a string representing a [time/date in ISO format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)
+
+Let's create the tickets in ERTP! The first step is to describe the **extents**. [Extents](https://agoric.com/documentation/ertp/guide/extent.html) are [comparable](https://github.com/Agoric/agoric-sdk/blob/f1b9dcae7d22aa7adc6222a42a0727ea29c055f3/packages/ERTP/util/sameStructure.js#L243-L245) values (as a first approximation, you can think of data-only objects that would translate as well in [JSON](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON)).
+
+```js
+const startDateString = (new Date(2019, 11, 9, 20, 30)).toISOString()
+
+const ticketExtents = Array(1114).fill().map((_, i) => ({
+  seat: i+1,
+  show: 'The Sofa',
+  start: startDateString,
+}))
+```
+
```

Comment:
Do we need to differentiate between extents and units here? Could we instead just focus on units? How about this:

```js
const startDateString = (new Date(2019, 11, 9, 20, 30)).toISOString()
const label = balletTicketMint.getAssay().getLabel();

const ticketUnits = Array(1114).fill().map((_, i) => ({
  label, 
  extent: {
    seat: i+1,
    show: 'The Sofa',
    start: startDateString,
  },
}))
```

Then we could just mention that extents are the non-labeled portion.

Reply:
> Could we instead just focus on units?

Now that i better understand units and why they exist, i think i agree
But i would remove the notion of extent altogether from this guide then. My belief is that the less notions introduced per article the better

In the same vein, I don't think showing the `{label, extent}` object provides any value to the reader
I would do:
```js
const ticketUnits = Array(1114).fill()
  .map((_, i) => balletTicketMint.getAssay().makeUnits({
    seat: i+1,
    show: 'The Sofa',
    start: startDateString,
  })
)
```

My belief on this is that people modeling ballet tickets want to model `{seat, show, start}` objects because it matches the mental model they have of what they're selling
And then they want to understand how to express this in the ERTP world

For this, they need to know about units because it's what mints take as an argument, and they need to know how to make units from the JavaScript objects they created, but they don't need to know the implementation details of units
So i'm leaving out the `{label, extent}` object from the code sample. Also, i think `makeUnits` is more explicit and more directly conveys the intention

Reply:
Ok, that sounds fair. I think `units` are mysterious to people until we break it down into a label + extent (hopefully in the future, an assay + extent), so if we can do that as well, that'd be great.

Reply:
I simplified this part by only talking about units. It also links to [units documentation](https://agoric.com/documentation/ertp/guide/units.html#labeled-extents) which explains units in terms for label+extent with a code example
This way, the guide can remain focused on the shortest path from "what the person wants to model" (`{seat, show, start}` objects) to "ERTP goods that can be bought/sold", but people who want to know have a simple way of knowing more about what units are and how they work